CmisSync.Lib/CmisSync.Lib.csproj

```diff
@@ -61,8 +61,13 @@
     <Reference Include="System.Data.SQLite">
       <HintPath>..\Extras\sqlite-netFx40-binary-Win32-2010-1.0.84.0\System.Data.SQLite.dll</HintPath>
     </Reference>
-    <Reference Include="Mono.Data.Sqlite" />
-    <Reference Include="Mono.Posix" />
+    <Reference Include="Mono.Data.Sqlite">
+      <HintPath>..\Extras\Mono.Data.Sqlite.dll</HintPath>

```

Comment:
Yes, I manualy and the file "Mono.Posix.dll" and "Mono.Data.Sqlite.dll" in Extras.
Below the file.
[Mono.posix_Mono.Data.Sqlite.zip](https://github.com/aegif/CmisSync/files/506430/Mono.posix_Mono.Data.Sqlite.zip)